flexmeasures/api/v3_0/assets.py

```diff
@@ -161,6 +171,22 @@ def index(
         query = query_assets_by_search_terms(
             search_terms=filter, filter_statement=filter_statement
         )
+
+        if sort_by is not None:
+            valid_sort_columns = {
+                "name": GenericAsset.name,
+                "owner": GenericAsset.account_id,
+            }
+
+            if sort_by not in valid_sort_columns:
+                return {"error": f"Invalid sort column: {sort_by}"}, 400
```

Comment:
This would be 422 code. Actually, the `sort_by` parameter could already be checked by MarshMallow (you can add valid values to the decorator I believe).

Reply:
ok, using the marshmallow approach could reduce the code in the API, I'll implement that